core/src/io/anuke/mindustry/world/blocks/distribution/Conduit.java

```diff
@@ -17,6 +17,8 @@
     protected TextureRegion[] topRegions = new TextureRegion[7];
     protected TextureRegion[] botRegions = new TextureRegion[7];
 
+    public float bar;
```

Comment:
This should be a property of `BlockStorage` to prevent an ugly cast, and should have a different name (like pressureScale or something).